config/ldp/metadata-parser/default.json

```diff
@@ -7,8 +7,19 @@
       "@type": "ParallelHandler",
       "handlers": [
         { "@type": "ContentTypeParser" },
-        { "@type": "LinkTypeParser" },
-        { "@type": "SlugParser" }
+        { "@type": "SlugParser" },
+        {
+          "@id": "urn:solid-server:default:TypeLinkParser",
+          "@type": "LinkParser",
+          "value": "type",
+          "predicate": "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
+        },
+        {
+          "@id": "urn:solid-server:default:InboxLinkParser",
+          "@type": "LinkParser",
+          "value": "http://www.w3.org/ns/ldp#inbox",
+          "predicate": "http://www.w3.org/ns/ldp#inbox"
+        }
```

Comment:
Would put them in a separate folder since they're bigger now. See the metadata-writer config for an example.

Reply:
Would you then also prefer to mirror the [A] naming and [B] config style of the `LinkRelMetadataWriter` (using one writer with a list of relations, instead of creating one for each relation)?

Reply:
Actually don't have a strong opinion about it, but now that you mention it it might be nice to stay parallel with that implementation indeed.